packages/ra-core/src/dataProvider/useGetOne.ts

```diff
@@ -16,11 +16,15 @@ import useDataProvider from './useDataProvider';
  * with the same parameters, until the response arrives.
  *
  * @param resource The resource name, e.g. 'posts'
- * @param id The resource identifier, e.g. 123
- * @param {Object} options Options object to pass to the react-query queryClient.
- * @param {boolean} options.enabled Flag to conditionally run the query. If it's false, the query will not run
- * @param {Function} options.onSuccess Side effect function to be executed upon success, e.g. { onSuccess: { refresh: true } }
- * @param {Function} options.onError Side effect function to be executed upon failure, e.g. { onError: error => notify(error.message) }
+ * @param {Params} params The getOne parameters { id }, e.g. { id: 123 }
+ * @param {Options} options Options object to pass to the react-query queryClient.
+ *
+ * @typedef Params
+ * @props params.id a resource identifier, e.g. 123
```

Comment:
```suggestion
 * @prop params.id A resource identifier, e.g. 123
```

Reply:
In fact, according to the jsDoc spec, it should be property... https://jsdoc.app/tags-typedef.html

Reply:
But it could be @prop as well [synonym](https://jsdoc.app/index.html#:~:text=%40property%20(synonyms%3A%20%40prop))

Reply:
Why do we need a @typedef when we already use TS? You can put those comments as jsdoc on the type and its properties directly, no?

Reply:
@WiXSL you're totally right, I'm pushing an update